IO.go

```diff
@@ -349,3 +349,8 @@ func iggIoSetClipboardText(handle C.IggIO, text *C.char) {
 	}
 	board.SetText(C.GoString(text))
 }
+
+// SetConfigDragClickToInputText [BETA] Enable turning DragXXX widgets into text input with a simple mouse click-release (without moving). Not desirable on devices without a keyboard.
+func (io IO) SetConfigDragClickToInputText(value bool) {
```

Comment:
Is this function mandatory? If it's marked "beta", then it might change, causing yet another major version bump for the wrapper.